src/crud/updateBackTranslationType.ts

```diff
@@ -0,0 +1,68 @@
+import { QueryBuilder, TransformBuilder } from '@orbit/data';
+import MemorySource from '@orbit/memory';
+import { AxiosError } from 'axios';
+import { ArtifactType, MediaFile } from '../model';
+import { UpdateRelatedRecord } from '../model/baseModel';
+import { logError, Severity } from '../utils';
+import { axiosGet } from '../utils/axios';
+
+export const updateBackTranslationType = async (
+  memory: MemorySource,
+  token: string | null,
+  user: string,
+  errorReporter: any
+) => {
+  if (token) {
+    await axiosGet('mediafiles/wbt', undefined, token)
+      // eslint-disable-next-line no-loop-func
+      .then((response) => {
+        var fr = response.data as any;
+        console.log(fr);
+      })
+      // eslint-disable-next-line no-loop-func
+      .catch((err: AxiosError) => {
+        logError(
+          Severity.error,
+          errorReporter,
+          'Whole Back Translate Update Failed:' +
+            (err.message || err.toString())
+        );
+      });
+  } else {
+    //offline
+    var artifacttypes = memory.cache.query((q: QueryBuilder) =>
+      q.findRecords('artifacttype')
+    ) as ArtifactType[];
+    var bt =
+      artifacttypes.filter((a) => a.attributes.typename === 'backtranslation')
+        .length > 0
+        ? artifacttypes.filter(
+            (a) => a.attributes.typename === 'backtranslation'
+          )[0]
+        : null;
```

Comment:
Would artifacttypes.find((a) => a.attributes.typename === 'backtranslation'); not be better? I think it returns the record or undefined.